### PR TITLE
[gitlab] Update periodic cleanup script to remove all kitchen resources

### PR DESCRIPTION
**Pull request by KSerrania**
### What does this PR do?

Updates the periodic cleanup script invocation to also clean up resources created by system-probe and security-agent kitchen tests.

### Motivation

system-probe and security-agent kitchen resources do not get periodically deleted.

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -4626,7 +4626,9 @@ periodic_kitchen_cleanup_azure:
     - export ARM_CLIENT_ID=`aws ssm get-parameter --region us-east-1 --name ci.datadog-agent.azure_client_id --with-decryption --query "Parameter.Value" --out text`
     - export ARM_CLIENT_SECRET=`aws ssm get-parameter --region us-east-1 --name ci.datadog-agent.azure_client_secret --with-decryption --query "Parameter.Value" --out text`
     - export ARM_TENANT_ID=`aws ssm get-parameter --region us-east-1 --name ci.datadog-agent.azure_tenant_id --with-decryption --query "Parameter.Value" --out text`
-    - python3.6 ~/deploy_scripts/cleanup_azure.py
+    - RESOURCE_GROUP_PREFIX=kitchen-dd-agent python3.6 ~/deploy_scripts/cleanup_azure.py
+    - RESOURCE_GROUP_PREFIX=kitchen-dd-security-agent python3.6 ~/deploy_scripts/cleanup_azure.py
+    - RESOURCE_GROUP_PREFIX=kitchen-dd-system-probe python3.6 ~/deploy_scripts/cleanup_azure.py
```
Comment (mx-psi) [inline .gitlab-ci.yml:4631]: Should we mention the need for updating this somewhere in the docs? Also, should #7075 add this for their tests?

**Review by mx-psi — APPROVED** [review]
🚀

File: .gitlab-ci.yml (hunk 1 above)
Comment (lebauce) [reply .gitlab-ci.yml:4631]: @mx-psi If this one is merged first, we'll update our PR accordingly

Comment (mx-psi) [reply .gitlab-ci.yml:4631]: Awesome, thanks!

Comment (KSerrania) [reply .gitlab-ci.yml:4631]: I don't think it needs to, the new test suite (`dd-security-agent-stress`) should already be covered by the `RESOURCE_GROUP_PREFIX=kitchen-dd-security-agent` line.

I'll add a line in the kitchen test setup docs to specify that this needs to be updated when a test suite with a new prefix is created.

**Review by mx-psi — APPROVED** [review]

**Pull request merged**